go.mod

```diff
@@ -35,26 +34,36 @@ require (
 // Pinned to kubernetes-1.16.2
```

Comment:
Can you validate whether this is still true? `k8s.io/kubernetes` has been removed from `require`, and `k8s.io/api` and others have been changed to `0.17.1`.

Reply:
`k8s.io/kubernetes` is an indirect dependency that needs not be there in the "require" section.
All the tests are passing. Not sure what another validation is required.